src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -332,42 +314,40 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
                 set_header: [Bounce.msgCount, bounceMessage]
                 regex_match: true
                 split_by: \s*(\d+)\s\s\s(.*.)
-```
-
-  
+```        
+    </Step>
+    <Step>
 ## Forward your Postfix logs to New Relic
-  
-You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward Postfix logs to New Relic.
 
-On Linux machines, your log file named `logging.yml` should be present in this path:
+1. Create a file named `logging.yml` in the infrastrucutre agent directory: 
 
 ```shell
-/etc/newrelic-infra/logging.d/
+touch /etc/newrelic-infra/logging.d/logging.yml
 ```
 
-After creating the log file, add the following script to the `logging.yml` file:
+2. Add the following snippet to the `logging.yml` file:
 
 ```yml
 logs:
   - name: mail.log
     file: /var/log/mail.log
     attributes:
       logtype: postfix_maillog
-```
+```        
+    </Step>
+    <Step>
+## Restart the infrastructure agent [#restart-infra]
 
-## Restart the New Relic infrastructure agent
-
-Before you can start reading your data, use the instructions in our [infrastructure agent docs](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent.
+Use the instructions in our [infrastructure agent docs](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/) to restart your infrastructure agent. This is a basic command that should work for most people:
 
 ```shell
 sudo systemctl restart newrelic-infra.service
-```
-In couple of minutes, your application will send metrics to [one.newrelic.com](https://one.newrelic.com).
-
-
+```            
+    </Step>
+    <Step>
 ##  Monitor your application
 
-You can choose our pre-built dashboard template named `Postfix` to monitor your Postfix application metrics. Follow these steps to use our pre-built dashboard template:
+Follow these steps to use our pre-built dashboard template to easily view your Postfix data:
 
 1. Go to <DoNotTranslate>**[one.newrelic.com](https://one.newrelic.com/)**</DoNotTranslate> and click on <DoNotTranslate>**+ Add data**</DoNotTranslate>.
```

Comment:
i'm just noticing this, but should these substeps be nested? i would go back and check sonarqube to make sure they are as well